src/elements/image/image.ts

```diff
@@ -437,7 +439,9 @@ export class SbbImageElement extends LitElement {
       imageUrlObj.searchParams.append('fp-debug', 'true');
     }
 
-    return imageUrlObj.href;
+    return isFullyQualifiedUrl
+      ? imageUrlObj.href
+      : imageUrlObj.pathname.substring(1) + imageUrlObj.search;
```

Comment:
```suggestion
    // In case of "noop" host, we don't return the host and must remove the starting `/` of the pathname.
    return isFullyQualifiedUrl
      ? imageUrlObj.href
      : imageUrlObj.pathname.substring(1) + imageUrlObj.search;
```